src/mozilla_sec_eia/models/sec10k/ex_21/inference.py

```diff
@@ -382,10 +374,10 @@ def postprocess(self, all_outputs):
         """Return logits, model predictions, and the extracted dataframe."""
         logits = all_outputs["logits"]
         predictions = all_outputs["logits"].argmax(-1).squeeze().tolist()
-        output_df = self.extract_table(all_outputs)
+        output_df = self._extract_table(all_outputs)
         return logits, predictions, output_df
 
-    def extract_table(self, all_outputs):
+    def _extract_table(self, all_outputs):
```

Comment:
I was getting confused between this `extract_table` function that's within the LayoutLM pipeline and the `extract_filings` function which orchestrates running the model and extracting a table so I stuck an underscore onto this one.